spacy_llm/backends/rest/backend/openai.py

```diff
@@ -87,6 +88,13 @@ def credentials(self) -> Dict[str, str]:
                 "(See OpenAI API documentation: https://platform.openai.com/docs/models/gpt-3)"
```

Comment:
I'm not sure that's the intended link

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/110/commits/3b234885621900b26f543c29920e522d8892d9b1.